src/shared-components/optionButton/__snapshots__/test.tsx.snap

```diff
@@ -214,19 +212,17 @@ exports[`<OptionButton /> UI snapshots radio unselected, with icon node prop 1`]
   fill: #332e54;
 }
 
-<div
+<button
   className="emotion-8 emotion-9"
   onClick={[Function]}
-  tabIndex="0"
-  type="clickable"
+  type="button"
 >
   <div
     className="emotion-6 emotion-7"
   >
     <div
       className="emotion-0 emotion-1"
       selected={false}
-      type="secondary"
```

Comment:
This isn't a regression: we currently pass `type` down as a DOM property even though we're only using it for styling.